src/components/texts/Date.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { StyleSheet } from 'react-native'
+import Text from './Text'
+import { theme } from '~/constants'
+
+const Date = (props) => {
```

Comment:
Are all dates rendered with the same style on the app? Why not just as text? I don't know, this component doesn't seem as it's really needed.

Reply:
Yes. Dates have a specific size and a specific font. While normal text uses `Red Hat Regular`, dates use `Red Hat Medium` and with a lower size.